_examples/storage/README.md

```diff
@@ -0,0 +1,98 @@
+
+
+Installation
+------------
+
+What you need? Obvously what you need is an *aerospike* server, the easiest way to having one for testing is running the **docker** container provided by Aerospike:
+```
+docker run -d -p 3000:3000 --name aerospike aerospike/aerospike-server
+```
+
+The next thing, is the example code:
+```
+go get -u github.com/mcuadros/go-git-aerospike/...
+```
+
+If you have the `GOPATH` on your `PATH`, you are ready to go, if now, this could be a good momento for add it. 
+
+
+Usage
+-----
+
+### Cloning the repository into the database
+
+Running the command with the `clone` option followed by the URL of a git repository like:
+```sh
+storage clone https://github.com/src-d/flamingo.git
+```
+
+the repository is stored on the aerospike database, this means that all the internal objects like commits, trees, blobs and tag are `records` in differents `set` in the `test` namespace:
+
+```sql
+aql> SELECT hash, type, url FROM test.commit
+```
+```
++--------------------------------------------+----------+-------+-----------------------------------+
+| hash                                       | type     | blob  | url                               |
++--------------------------------------------+----------+-------+-----------------------------------+
+| "c94450c805876e49b38d2ff1103b8c09cdd2aef4" | "commit" | 00 00 | ...github.com/src-d/flamingo.git" |
+| "7f71640877608ee9cfe584fac216f03f9aebb523" | "commit" | 00 00 | ...github.com/src-d/flamingo.git" |
+| "255f097450dd91812c4eb7b9e0d3a4f034f2acaf" | "commit" | 00 00 | ...github.com/src-d/flamingo.git" |
++--------------------------------------------+----------+-------+-----------------------------------+
+102 rows in set (0.071 secs)
+```
+
+And also the references and the configuration (remotes) are stored on it.
+
+```sql
+aql> SELECT name, target, url FROM test.reference
+```
+```
++------------------------------+--------------------------------------------+-----------------------+
+| name                         | target                                     | url                   |
++------------------------------+--------------------------------------------+-----------------------+
+| "HEAD"                       | "ref: refs/heads/master"                   | ...rc-d/flamingo.git" |
+| "refs/heads/master"          | "ed3e1aa2e46584cb803ed356cb5d8855f6d05660" | ...rc-d/flamingo.git" |
+| "refs/remotes/origin/master" | "ed3e1aa2e46584cb803ed356cb5d8855f6d05660" | ...rc-d/flamingo.git" |
++------------------------------+--------------------------------------------+-----------------------+
+3 rows in set (0.046 secs)
+```
+
+### Reading the repository
+
+If now we run the command with the option `log`:
```

Comment:
Reword:

> Running the `log` command:

This is a good place to explain the why we need to pass the url and that this needs to be done after cloning.